### PR TITLE
Avoid dropping _cache_card until after a database update is completed.

### LINKED ISSUE #6345: We get little bursts (daily, all at the same time) of 500 errors because "table _cache_card does not exist". This should not happen
Reported on Discord by bakert#2193

### PR DESCRIPTION
**Pull request by bakert**
This will prevent the site from going down for a couple of minutes during
a scryfall update (which are daily when spoilers for new sets are being
releaesed).

Fixes #6345.

### CODE REVIEW

**Comment by bakert** [comment]
I'm adding "do not merge" label here because this makes me nervous. Local testing says it's fine but want someone to say "I can't see anything obviously terrible here" before I merge :D @silasary 
?

**Comment by tsutton** [comment]
I tested this a bit locally, including having the update code pause at some key points and making sure the website still worked, and it all looks good to me.

**Review by tsutton — APPROVED** [review]

**Pull request merged**
